index.bs

```diff
@@ -509,8 +509,13 @@ Each {{console}} namespace object has an associated <dfn>timer table</dfn>, whic
 
 <h4 id="time" oldids="time-label,dom-console-time" method for="console">time(<var>label</var>)</h4>
 
-<a for="map">Set</a> the value of the entry with key <var>label</var> in the associated
-<a>timer table</a> to the current time.
+<emu-alg>
+  1. If the associated <a>timer table</a> contains an entry with key _label_, abort, optionally
+     reporting a warning to the console indicating that a timer with label _label_ has already been
+     started.
+  1. Otherwise, <a for="map">set</a> the value of the entry with key _label_ in the associated
+     <a>timer table</a> to the current time in an implementation-defined format.
```

Comment:
I'm not sure we need "in an implementation-defined format" since this is a time, not a string.